vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileFragment.kt

```diff
@@ -208,9 +210,10 @@ class RoomProfileFragment @Inject constructor(
                 Timber.w("The room has been left")
                 activity?.finish()
             } else {
+                val roomType = RoomUtils.getRoomType(it)
                 headerViews.roomProfileNameView.text = it.displayName
                 views.matrixProfileToolbarTitleView.text = it.displayName
-                headerViews.roomProfileAliasView.setTextOrHide(it.canonicalAlias)
+                headerViews.roomProfileAliasView.setTextOrHide(it.canonicalAlias.takeIf { roomType != TchapRoomType.FORUM })
```

Comment:
`headerViews.roomProfileAliasView` should be hidden currently in all cases in Tchap